Altis_Life.Altis/stringtable.xml

```diff
@@ -1307,7 +1307,7 @@
             <Spanish>Your message cannot exceed 400 characters!</Spanish>
             <Russian>Your message cannot exceed 400 characters!</Russian>
             <German>Deine Nachricht darf maximal 400 Zeichen enthalten!!</German>
-            <French>Votre message ne peut pas dépasser 400 caractères !</French>
+            <French>Votre message ne peut pas excéder 400 caractères!</French>
```

Comment:
We need a space before the "!" :) 
[Source](https://github.com/AsYetUntitled/Framework/pull/179#issuecomment-262789117)